src/MusicStore/Program.cs

```diff
@@ -7,6 +7,7 @@ public static class Program
         public static void Main(string[] args)
         {
             var host = new WebHostBuilder()
+                .UseUrls("http://+:5000")
```

Comment:
This should be set through environment variables and not in the app
I would recommend we stick with command line `server.urls=http://*:5000` not change the app to expose publicly